schedule/yast/skip_registration/skip_registration_15sp3.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name:           skip_registration
+description:    >
+  Skipping registration for SLE 15, as requires network connection.
+  This is default behavior for SLE 12.
+vars:
+  SCC_REGISTER: 'none'
+  ADDONS: 'desktop'
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
```

Comment:
we use schedules based on default, we don't use list format anymore for new test suites, could you please convert it, and additionally remove the vars, which we have moved to job group yaml. I'm not sure we need all those addons above...

Reply:
'schedule/yast/skip_registration/skip_registration.yaml' was used by the rest of the maintenance skip_registration tests  (15 SP4 and 15 SP5) to avoid creating new files. Since we need to create a new schedule for SP3, then we can create new schedules for the rest of them as well in the name of consistency, but I would recommend to do it in a separate ticket.